src/vs/workbench/contrib/bulkEdit/browser/preview/bulkEditPreview.ts

```diff
@@ -316,7 +316,11 @@ export class BulkFileOperations {
 				let ignoreAll = false;
 
 				for (const edit of file.originalEdits.values()) {
-					if (edit instanceof ResourceTextEdit) {
+					if (edit instanceof ResourceFileEdit) {
+						const content = await edit.options.contents;
```

Comment:
Can we do the awaits in parallel? (Promise.all)

Reply:
Fair point. PTAL